region/lowernorfair/west/Screw Attack Room.json

```diff
@@ -611,6 +611,131 @@
       ],
       "endsWithShineCharge": true
     },
+    {
+      "link": [1, 4],
+      "name": "Side Platform Jump with Blue Speed",
+      "entranceCondition": {
+        "comeInWithSidePlatform": {
+          "platforms": [
+            {
+              "minHeight": 1,
+              "maxHeight": 1,
+              "minTiles": 17,
+              "speedBooster": true,
+              "obstructions": [[1, 0]],
+              "requires": [
+                {"getBlueSpeed": {
+                  "usedTiles": 15,
+                  "openEnd": 1
+                }},
+                {"or": [
+                  "canMomentumConservingMorph",
+                  {"and": [
+                    "canBeVeryPatient",
+                    "canInsaneJump"
+                  ]}
+                ]},
+                {"heatFrames": 80}
+              ],
+              "note": ["This applies to Warehouse Entrance."]
+            },
+            {
+              "minHeight": 2,
+              "maxHeight": 2,
+              "minTiles": 27.4375,
+              "speedBooster": true,
+              "obstructions": [[1, 0]],
+              "requires": [
+                {"getBlueSpeed": {
+                  "usedTiles": 24,
+                  "openEnd": 1
+                }},
+                {"heatFrames": 60}
+              ],
+              "note": ["This applies to Dust Torizo Room."]
+            },
+            {
+              "minHeight": 2,
+              "maxHeight": 2,
+              "minTiles": 27.4375,
+              "speedBooster": true,
+              "obstructions": [[1, 0]],
+              "requires": [
+                {"getBlueSpeed": {
+                  "usedTiles": 15,
+                  "openEnd": 1
+                }},
+                "canLateralMidAirMorph",
+                {"heatFrames": 70}
+              ],
+              "note": [
+                "This applies to Noob Bridge.",
+                "Use a 2-tap in order to gain blue speed with high momentum."
+              ],
+              "detailNote": [
+                "The specific speed $3.2 does not work as Samus will not get high enough to break the blocks.",
+                "A lower speed of $3.1 can work, but it is recommended to instead do a double-stutter with an early second tap,",
+                "to more reliably gain a higher speed of $3.3 or higher."
+              ]
+            },
+            {
+              "minHeight": 3,
+              "maxHeight": 3,
+              "minTiles": 37.4375,
+              "speedBooster": true,
+              "obstructions": [[3, 0]],
+              "requires": [
+                "canMomentumConservingMorph",
```

Comment:
Works for placement.  I didn't touch the overhang most of the time.  It's hard to describe how to best do the 2 tap or if its even better than full speed.